cmd/pipeline/deploy.go

```diff
@@ -375,7 +421,7 @@ func (o *DeployOptions) setDefaults() error {
 }
 
 func (o *DeployOptions) toPipelineDeployClientOptions() (types.PipelineDeployOptions, error) {
-	varList := []types.Variable{}
+	var varList []types.Variable
```

Comment:
Change suggested by the linter ✨

Reply:
Which linter are you using? I think we should all start using the same linter to have a common agreement on these things

Reply:
@jLopezbarb it's the one configured in the PR actions, DeepSource: https://github.com/okteto/okteto/pull/3455/checks?check_run_id=12068422538 

Does it block only my PRs? 😂

Reply:
I guess the confusion might be the word "linter" - this seems more like a generic code health solution, but this sort of suggestions generally come from linters - sorry for the confusion :)